### ISSUE TITLE
Support for mouse/magic remote

### ISSUE THREAD

**Issue by kirandcunha** [rdkcentral/Lightning#68]
Hi Team,
Is there a support for mouse/magic remote available here? I did not find any reference in the documentation.

**Comment by erikhaandrikman** [comment]
There is currently no support for mouse / magic remote. Magic remote support is on our roadmap

**Comment by markus101** [comment]
I've taken a look at the branch that starts support for this (https://github.com/WebPlatformForEmbedded/Lightning/tree/touch-support), but it is missing handling for the `enableTouch` option as it's never added to the app options via `_setOptions`. 

Once I fixed the options locally I tested on webOS and found touch support is not sufficient for magic remote support as the magic remote behaves like a regular mouse. Once things were changed to use `mousedown` instead of `touchstart` it worked as expected on webOS.
